### PR TITLE
DDCYLS-4987

### PR DESCRIPTION
**Pull request by baha-aboukoura**
# DDCYLS-4987 - upgrade to Play 3.0

**Upgrade**

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs and context.

## Checklist

* [ ] I've included appropriate tests with any code I've added
* [x] I've executed the acceptance test pack locally to ensure there are no regressions
* [x] I've added my code using logical, atomic commits, squashing as appropriate - including the Jira issue number in the commit message
* [x] I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

**Comment by baha-aboukoura** [comment]
Unit tests passed
Screenshot 2024-04-09 at 18 04 38

**Comment by baha-aboukoura** [comment]
integration tests passed
Screenshot 2024-04-09 at 18 06 36

**Comment by baha-aboukoura** [comment]
a11y tests passed
Screenshot 2024-04-09 at 18 10 26

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-addSbtPlugin("uk.gov.hmrc" % "sbt-accessibility-linter" % "0.33.0")
-addSbtPlugin("uk.gov.hmrc" %% "sbt-auto-build" % "3.16.0") //cannot go any higher on Scala 2.12
-addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.2.0")
-addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
-addSbtPlugin("org.scoverage" %% "sbt-scoverage" % "1.9.3")
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.18")
-addSbtPlugin("io.github.irundaia" % "sbt-sassify" % "1.5.2")
\ No newline at end of file
+// To resolve a bug with version 2.x.x of the scoverage plugin - https://github.com/sbt/sbt/issues/6997
+ThisBuild / libraryDependencySchemes ++= Seq("org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always)
```
Comment (tapiwa-tiyemba) [inline project/plugins.sbt:5]: is this needed ?

Comment (baha-aboukoura) [reply project/plugins.sbt:5]: good catch it's no longer required

Comment (baha-aboukoura) [reply project/plugins.sbt]: apparently it's still needed https://build.tax.service.gov.uk/job/BTA/job/VAT%20Registration/job/vat-registration-eligibility-frontend-pr-builder/58/console

Comment (baha-aboukoura) [reply project/plugins.sbt]: but for some reason it works locally

**Review by tapiwa-tiyemba — APPROVED** [review]

**Pull request merged**